CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ This changelog keeps track of all changes to the packs-sdk. We follow convention
 - **Breaking Change** Removed dead code related to `defaultConnectionType`. This was an unused feature and should not have been previously referenced.
 - **Future Breaking Change** `SetEndpoint.getOptionsFormula` has been renamed `SetEndpoint.getOptions` for clarity.
 - **Future Breaking Change** The `attribution` property is moving from being a child field on `identity` within an object schema to just being a top-level field on the object schema.
+- **Future Breaking Change** The `defaultValue` property of parameter definitions will be renamed to `suggestedValue` to reflect that these are values that do not act as a true default but rather prepopulates a value when used. During a migration period, you may use either name; the original name is now marked as deprecated and will be removed at a future date, no earlier than May 1, 2022.
```

Comment:
Nit, can hold off on the May 1 language for now, since we'll bundle all these changes together we can put a date on all of them at once.